packages/flutter/lib/src/services/keyboard_maps.dart

```diff
@@ -1176,6 +1177,196 @@ const Map<int, LogicalKeyboardKey> kMacOsFunctionKeyMap = <int, LogicalKeyboardK
   0x0000005a: LogicalKeyboardKey.f20,
 };
 
+/// Maps iOS-specific key code values representing [PhysicalKeyboardKey].
+///
+/// iOS doesn't provide a scan code, but a virtual keycode to represent a physical key.
+const Map<int, PhysicalKeyboardKey> kIOSToPhysicalKey = <int, PhysicalKeyboardKey>{
```

Comment:
Naming conventions say this should be called `kIosToPhysicalKey`. (The style guide actually recommends not starting class names with iOS, and using "Cupertino" or "UiKit" in order to avoid the weird capitalization, but in this case, we're referring to the platform itself, not the UI toolkit or widget set, so I think it is unavoidable. It still needs to be named consistent with other constants, though.)